headjack/models/utterance.py

```diff
@@ -49,6 +42,27 @@ def convo(
         history = history[::-1]
         return "\n".join(str(u) for u in history) + "\n"
 
+    @property
+    def log_str(self) -> str:
+        """Creates a string for logging the utterance"""
+        return "Utterance: " + json.dumps(
```

Comment:
Is the `"Utterance: "` needed? The logs may be easier to process downstream if they're just line separated valid json objects.

Reply:
I guess the motivation was so you could use a pattern like `^Utterance:.*$` to get the lines but if you just get the json lines that have particular keys that's probably enough so I'll get rid of the 'Utterance:'